src/main/resources/webroot/index.html

```diff
@@ -193,6 +97,11 @@
   </div>
 </header>
 <div class="container" ng-controller="VertxStarterController as vm">
+  <a ng-click="vm.goToGenerateButton()" class="go-to-generate" style="display: inline;">
```

Comment:
I was wondering what this does then realized the arrow down image was displayed on the bottom right of the page, even when everything fits into the viewport :smile: (Firefox 68 on Fedora 29)

Is it possible to:

- hide it when the dependencies panel is folded?
- display it on the top right of the viewport?

Reply:
Just tried the code, I believe this is the only outstanding issue.

Reply:
I went ahead and merge the branch without the "go to generate" button. Feel free to submit another PR to address the comment above.